thoth/adviser/steps/security_indicators.py

```diff
@@ -121,6 +122,11 @@ def run(
                 package_version=package_version.locked_version,
                 index_url=package_version.index.url,
             )
+            msg = (
+                f"Thoth has security info for {package_version.name}==={package_version.locked_version} "
+                f"on {package_version.index_url}"
+            )
+            self.context.stack_info.append({"type": "info", "message": msg})
```

Comment:
This should go to the justification returned rather than to `stack_info`. `stack_info` is generic for the whole application stack and is not specific to resolved stack. By adding this message to stack info, it can happen that the recommender recommends a different stack without the reported package (which will be misleading). Here and below.

It could be also good to [provide a link to justification](https://thoth-station.ninja/docs/developers/adviser/justifications.html#creating-justification-documents).